lib/application_responder.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ApplicationResponder < ActionController::Responder
+  include Responders::FlashResponder
+  include Responders::HttpCacheResponder
+
+  # Redirects resources to the collection path (index action) instead
+  # of the resource path (show action) for POST/PUT/DELETE requests.
+  # include Responders::CollectionResponder
```

Comment:
do we need this comment and commented line. maybe delete?

Reply:
This file was automatically created by a generator for the `responders` gem. I assume we won't need this bit... :ok_hand: